python/py-spdx-lookup/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-spdx-lookup
+version             0.3.2
+revision            0
+
+maintainers         nomaintainer
+description         SPDX license list query tool
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://github.com/bbqsrc/spdx-lookup-python
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             2-clause BSD License
+checksums           sha256  d81e16ace58f668be3c2a8502355269e1661083a0c8538015945821469f6a5fa \
+                    rmd160  3ba50ad1b5d273d5c8e426d4d8375a3e2b6e5a56 \
+                    size    4763
+                                
```

Comment:
please don't leave trailing spaces on empty lines (also on line 32)

Reply:
Sorry, I will take care of this from now on.